cirq-ft/cirq_ft/algos/selected_majorana_fermion.py

```diff
@@ -43,8 +43,13 @@ class SelectedMajoranaFermionGate(unary_iteration_gate.UnaryIterationGate):
         See Fig 9 of https://arxiv.org/abs/1805.03662 for more details.
     """
 
-    selection_regs: infra.SelectionRegisters
-    control_regs: infra.Registers = infra.Registers.build(control=1)
+    selection_regs: Tuple[infra.SelectionRegister, ...] = attr.field(
+        converter=lambda v: (v,) if isinstance(v, infra.SelectionRegister) else tuple(v)
```

Comment:
elsewhere we use `cirq_ft.infra.type_convertors.to_tuple` (which I'm not a fan of, btw). Why is mypy happy here but not elsewhere?

side note: there's a spelling error: "converters"

Reply:
Not sure, but looks like `to_tuple` is also not needed anymore so I deleted the `type_convertors` module and replaced all usages with lambda expression. 

My guess would that we are probably using a newer mypy version now, but I'm not sure.